core/common/src/main/java/alluxio/Configuration.java

```diff
@@ -366,6 +366,24 @@ public static long getBytes(PropertyKey key) {
   }
 
   /**
+   * Gets the time in the given key.
+   *
+   * @param targetUnit the target unit set by user
+   * @param key the key to get the value for
+   * @return the time transformed from milliseconds to the target unit
+   */
+  public static double getTime(String targetUnit, PropertyKey key) {
+    String rawValue = get(key);
+
+    try {
+      return FormatUtils.parseTimeUnit(targetUnit,rawValue);
+    } catch (Exception ex) {
```

Comment:
nit: in our code convention, Exception variable is typically named after `e` rather than `ex`